src/components/Slideshow.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+    <div class="slideshow" :class="{'is-visible' : slideshowIsVisible}">
+        <div class="slideshow--screenshot">
+            <a href="" class="slideshow--close" v-on:click="onClose">
+                <Icon :icon="'close'" class="slideshow--close-icon"></Icon>
+            </a>
+            <img :src="`screenshots/${currentTestcase.screenshotFilename}`" :alt="currentTestcase.screenshotFilename">
+        </div>
+        <a href="" v-on:click="changePosition(-1, 0, $event)" class="slideshow--controls-left">
+            <Icon :icon="'left'" class="slideshow--controls-left-icon"></Icon>
+        </a>
+        <a href="" v-on:click="changePosition(0, -1, $event)" class="slideshow--controls-up">
+            <Icon :icon="'up'" class="slideshow--controls-up-icon"></Icon>
+        </a>
+        <a href="" v-on:click="changePosition(0, 1, $event)" class="slideshow--controls-down">
+            <Icon :icon="'down'" class="slideshow--controls-down-icon"></Icon>
+        </a>
+        <a href="" v-on:click="changePosition(1, 0, $event)" class="slideshow--controls-right">
+            <Icon :icon="'right'" class="slideshow--controls-right-icon"></Icon>
+        </a>
+    </div>
+</template>
+
+<script>
+	import { computed, reactive } from '@vue/composition-api';
+	import Icon from "@/components/Icon";
+
+	import SlideshowPosition from "@/model/slideshowPosition";
+
+	export default {
+		name: "Slideshow",
+		components: { Icon },
+		props: {
+			slideshowIsVisible: Boolean,
+			slideshowPosition: SlideshowPosition,
+			grid: Array
+		},
+		setup( props, { emit } ) {
+			const onClose = function( e ) {
+				e.preventDefault();
+				emit( 'close-slideshow' );
+			}
+
+			const currentSlideshowPosition = reactive( props.slideshowPosition );
```

Comment:
@gbirke You know the best way to set this to update whenever the prop changes? I tried to use `watchEffect` but that stops the navigation from working.